v1.14/oraclelinux/Dockerfile

```diff
@@ -1,15 +1,18 @@
-FROM container-registry.oracle.com/os/oraclelinux:7-slim
+FROM container-registry.oracle.com/os/oraclelinux:8-slim
```

Comment:
Should this image be coming from ghcr? `ghcr.io/oracle/oraclelinux:8-slim`

Reply:
We are generally pulling from oracle, I can toggle to ghcr